src/atldld/config.py

```diff
@@ -0,0 +1,47 @@
+# The package atldld is a tool to download atlas data.
+#
+# Copyright (C) 2021 EPFL/Blue Brain Project
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""Configuration of the atldld CLI app and library."""
+import os
+import pathlib
+
+import appdirs
+
+
+def user_cache_dir(create: bool = True) -> pathlib.Path:
+    """Determine currently configured cache directory for atldld.
+
+    Parameters
+    ----------
+    create
```

Comment:
Sorry, I am always confused about this! Shouldn't we use either the `type` or `name : type` syntax ?

Reply:
It's different for "Parameters" and "Returns", NumPy docstyle is really stupid in this one.
* For parameters: `name` or `name : type` [ref](https://numpydoc.readthedocs.io/en/latest/format.html#parameters)
* For returns: `type` or `name : type` [ref](https://numpydoc.readthedocs.io/en/latest/format.html#returns)

🤦🏻

Reply:
I think I will put a note on my computer for this, then! 😅 Thanks for the explanation!